haskell-language-server.cabal

```diff
@@ -1914,19 +1840,19 @@ executable haskell-language-server-wrapper
           unix
         , containers
 
-  default-language: Haskell2010
 
 
 test-suite func-test
-  import:             common-deps
+  import:             defaults
+                    , test-defaults
```

Comment:
You can remove "-threaded -rtsopts -with-rtsopts=-N" from func-test now that you're importing test-defaults